db/migrate/20230928220752_add_links_to_elections.rb

```diff
@@ -0,0 +1,6 @@
+class AddLinksToElections < ActiveRecord::Migration[7.0]
+  def change
+    add_column :elections, :terms_and_conditions, :string
+    add_column :elections, :learn_more, :string
```

Comment:
In case this is a URL:
```suggestion
    add_column :elections, :terms_and_conditions_url, :string
    add_column :elections, :learn_more_url, :string
```